### ISSUE TITLE
await on literal value causes error

### ISSUE THREAD

**Issue by otakustay** [MatAtBread/fast-async#45]
In such code:

```
const getSearchCount = () => {
  return await {
    repoCount: 100,
    codeCount: 20
  };
};
```

fast-async will generate wrong code such as (copied from console):

```
function getSearchCount(keyword) {
  return new Promise(function ($return, $error) {
    return {
      repoCount: 100,
      codeCount: 20
    }.then($return, $error);
  }.$asyncbind(this));
}
```

which obviously leads to error (object does not have `then` method)

**Comment by matAtWork** [comment]
This is documented in the README for nodent (which underlies fast-async). Look at the section "await non-Promise" inhttps://github.com/MatAtBread/nodent#gotchas-and-es7-compatibility.

TLDR; you need to set `wrapAwait: true` option.

**Comment by matAtWork** [comment]
Incidentally, in your example, getSearchCount is not marked as `async`, which works in `nodent`, but not `fast-async`, since it is an extension of the specification supported by nodent.
